plugins/ext_test/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
```

Comment:
I'd recommend adding a README.md in the **plugins** directory to just give some basic information on:
- What a `cmd2` plugin is
- How to use a `cmd2` plugin
- How to create your own `cmd2` plugin

It can be brief as long as it provides links to where to get more info.

Reply:
@kotfu has a plugin template in a separate repo with detailed instructions. I think maybe it makes sense to go ahead and bring that in here as well.

Reply:
Yeah it would be reasonable to do that.

Reply:
Yeah, I think me plan is to leave this directory as-is for this PR. Next, bring @kotfu's plugin template in in a subsequent PR and link up the tests and coverage and add a README here.